dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/NameAliases.java

```diff
@@ -49,6 +49,7 @@ public void newAliases(Map<String, String> newAliases) {
 
     public void resetAliases() {
         aliases = originalNames.clone();
+        aliasesExtensions.clear();
```

Comment:
need to also clear originalExtensions.

Please also add test cases for clearing/setting ext_ aliases to the json logging bucket - seems like this may be lacking